################################################################################
SHORT DESCRIPTION: 
################################################################################
Allows tpm_startup() function to run even if Xen is detected.

################################################################################
LONG DESCRIPTION: 
################################################################################
Allows tpm_startup() function to run even if Xen is detected.

################################################################################
CHANGELOG 
################################################################################

################################################################################
REMOVAL 
################################################################################
This patch can be removed safely if the feature is not wanted anymore.

################################################################################
UPSTREAM PLAN 
################################################################################
This is OpenXT specific. There is no plan to upstream this patch.

################################################################################
INTERNAL DEPENDENCIES 
################################################################################
None.

################################################################################
PATCHES
################################################################################
Index: seabios-1.9.1/src/tcgbios.c
===================================================================
--- seabios-1.9.1.orig/src/tcgbios.c
+++ seabios-1.9.1/src/tcgbios.c
@@ -509,9 +509,6 @@
         return;
 
     tpm_acpi_init();
-    if (runningOnXen())
-        return;
-
     tpm_startup();
 }

